RELEASE.md

```diff
@@ -1,10 +1,24 @@
 # Upcoming release:
+
 Add release notes for the upcoming release here.
 
+# Release 3.1.0:
+
 ## Major features and improvements
 
-- renamed `get_data_from_kedro` to `format_pipeline_data` (#72)
-- Added pipeline, and catalog arguments to `format_pipeline_data` (#72)
+- Allow users to export graph as a transparent SVG/PNG image (#82)
+- Add theme prop and icon button visibility prop (#80)
+- Rename `get_data_from_kedro` to `format_pipeline_data` (#72)
+- Add pipeline and catalog arguments to `format_pipeline_data` (#72)
```

Comment:
Aren't these 2 breaking changes?

Reply:
(discussed this with Lorena IRL)

We don't think it is, because these appear to be just private methods. But it'd be great to discuss this with @DmitriiDeriabinQB when he's back.